retail/interactive-tutorial/Snippets/CreateTestResources.cs

```diff
@@ -0,0 +1,371 @@
+﻿// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Storage.v1.Data;
+using Google.Cloud.Retail.V2;
+using Google.Cloud.Storage.V1;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+
+/// <summary>
+/// Class that performs creeation of all necessary test resources.
+/// </summary>
+public static class CreateTestResources
+{
+    private const string ProductFileName = "products.json";
+    private const string ProductsObject = "products.json";
+
+    private const string EventsFileName = "user_events.json";
+
+    private const string DataSetId = "products";
+    private const string TableId = "products";
```

Comment:
you have defined variable ProductTable with the same value and the same meaning. please remove this one , as I think the ProductTable is better name